dev/com.ibm.ws.opentracing_fat/publish/servers/opentracingFATServer1/server.xml

```diff
@@ -3,6 +3,7 @@
   <featureManager>
     <feature>servlet-3.1</feature>
     <feature>opentracing-1.0</feature>
+    <feature>jaxrs-2.0</feature>
```

Comment:
I'm not sure this is necessary - I thought `opentracing-1.0` already depended on `jaxrs-2.0`